api/prod-compose.yml

```diff
@@ -1,10 +1,14 @@
 services:
   api:
     restart: always
+    ports:
+      - '8039:8039'
+      - '8038:8080'
     expose:
-      - 8039
+      - '8039'
     environment:
       NODE_ENV: production
+      OVERWRITE_CATALOG: 'true'
```

Comment:
Does it not work without? This thing seems to be the route handler, which means it's run on every request, right?

Reply:
You're right, it works without since the container is recreated with a new static every time. I will add overwrite catalog as a bash argument to ./start.sh in dev since there may be times where we want to update catalogs in the mounted filesystem approach in dev.